xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -188,6 +189,8 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByArtist, sortAttribute, 557, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));  // Filename, empty | Userdefined, Userdefined
       // year
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));
+      // album date added
+      AddSortMethod(SortByDateAdded, sortAttribute, 570, LABEL_MASKS("%F", "", strAlbumLeft + " - " + strAlbumRight, "%a"));  // Filename, empty | Userdefined - Userdefined, dateAdded
```

Comment:
Can do, but it feels kind of awkward, when you don't see the date added.
Especially when you go from:
Artist (you see the date added) -> Album (you don't see it) -> Song (you see the date added)